portfolio/src/main/webapp/style.css

```diff
@@ -75,14 +75,14 @@ body {
   padding: 15px;
 }
 
```

Comment:
Remove leading 0 on line 63 (GitHub doesn't let me comment that far away from your changes). See https://google.github.io/styleguide/htmlcssguide.html#Leading_0s

Reply:
Made Changes